events/wc_court_events.txt

```diff
@@ -174,14 +174,45 @@ character_event = {
 	is_triggered_only = yes
 	
 	only_playable = yes
-	min_age = 16
+	min_age = 50
 	prisoner = no
 	
 	trigger = {
+		mercenary = no
+		holy_order = no
+		NOT = { is_offmap_governor = yes }
+		is_theocracy = no
+		OR = {
+			is_republic = no
+			government = merchant_republic_government
```

Comment:
Since there may be more governments that work like `merchant_republic_government`.
```suggestion
			is_patrician = yes
```

Reply:
fxd